force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,6 +40,12 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
+        String r = 'Hello, Salesforce!';
+        List<String> result = new List<String>();
+        String length = r.length().format();
+        result.add(length);
+        result.add(r.left(5));
+        result.add(r.toLowerCase());
```

Comment:
definitely name Hello Salesforce like `greeting` or `message` in a real world scenario, but this is fine because it's just a practice Method

Reply:
I would also make `result` plural `results`